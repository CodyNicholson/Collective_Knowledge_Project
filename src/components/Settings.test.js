//This file was begun using code from Link.react.test.js, from the Jest Testing React Apps Guide
//https://facebook.github.io/jest/docs/en/tutorial-react.html
import React from 'react';
import { Settings } from './Settings';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';


test('Tags module test', () => {
  const component = renderer.create(
    <MemoryRouter>
    	<Settings>
    	</Settings>
    </MemoryRouter>
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();

});