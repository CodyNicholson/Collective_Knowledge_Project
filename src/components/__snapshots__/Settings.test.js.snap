// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tags module test 1`] = `
<div
  className="settings-page"
>
  <div
    className="container page"
  >
    <div
      className="row"
    >
      <div
        className="col-md-6 offset-md-3 col-xs-12"
      >
        <h1
          className="text-xs-center"
        >
          Your Settings
        </h1>
        <form
          onSubmit={[Function]}
        >
          <fieldset>
            <fieldset
              className="form-group"
            >
              <input
                className="form-control"
                onChange={[Function]}
                placeholder="URL of profile picture"
                type="text"
                value=""
              />
            </fieldset>
            <fieldset
              className="form-group"
            >
              <input
                className="form-control form-control-lg"
                onChange={[Function]}
                placeholder="Username"
                type="text"
                value=""
              />
            </fieldset>
            <fieldset
              className="form-group"
            >
              <textarea
                className="form-control form-control-lg"
                onChange={[Function]}
                placeholder="Short bio about you"
                rows="8"
                value=""
              />
            </fieldset>
            <fieldset
              className="form-group"
            >
              <input
                className="form-control form-control-lg"
                onChange={[Function]}
                placeholder="Email"
                type="email"
                value=""
              />
            </fieldset>
            <fieldset
              className="form-group"
            >
              <input
                className="form-control form-control-lg"
                onChange={[Function]}
                placeholder="New Password"
                type="password"
                value=""
              />
            </fieldset>
            <button
              className="btn btn-lg btn-primary pull-xs-right"
              disabled={undefined}
              type="submit"
            >
              Update Settings
            </button>
          </fieldset>
        </form>
        <hr />
        <button
          className="btn btn-outline-danger"
          onClick={undefined}
        >
          Delete Profile
        </button>
        <button
          className="btn btn-outline-danger marginL"
          onClick={undefined}
        >
          Click here to logout.
        </button>
      </div>
    </div>
  </div>
</div>
`;
